modules/API/pages/gsql-endpoints.adoc

```diff
@@ -4162,3 +4162,2356 @@ Response::
 ====
 
 '''
+
+=== get loading job names
+
+`GET /gsql/v1/loading-jobs?graph=[graphName]`
+
+This endpoint is used to get all loading job names of a graph.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X GET 'http://localhost:14240/gsql/v1/loading-jobs?graph=g' 
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"","jobNames":["incidents_fraud_report_company_csv"]}
+----
+--
+====
+===== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| the loading job names of which graph
+|===
+
+'''
+
+=== create/update a loading job
+
+`POST/PUT /gsql/v1/loading-jobs?graph=[graphName]`

```

Comment:
Split this into 2 separate lines like how restpp built-in endpoints did: https://github.com/tigergraph/server-docs/pull/478/files.

We may want to avoid '/' in methods cause there are already lots of '/' in uri.